engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -158,11 +161,17 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
         this.renderingConfig = context.get(Config.class).getRendering();
         this.renderingDebugConfig = renderingConfig.getDebug();
         this.shaderManager = context.get(ShaderManager.class);
-
-        if (renderingConfig.isOculusVrSupport()) {
-            playerCamera = new OculusStereoCamera();
-            currentRenderingStage = RenderingStage.LEFT_EYE;
-
+        if (renderingConfig.isVrSupport()) {
+            this.vrProvider = new OpenVRProvider();
+            if (this.vrProvider.init()) {
+                playerCamera = new OpenVRStereoCamera(this.vrProvider);
+                currentRenderingStage = RenderingStage.LEFT_EYE;
+                context.put(OpenVRProvider.class, vrProvider);
+            }
+            else {
```

Comment:
This should be `} else {`.

Also, in line with previous comments, if isVrSupport() returns true and this.vrProvider.init() doesn't work, returning false, some kind of logger error/warning would help, i.e. stating that MONO mode is enabled despite vrSupport being ON. Or is this logged elsewhere?